src/test/unit/swagger/fetch-swagger.test.ts

```
@@ -0,0 +1,113 @@
+    mockFetch.mockResolvedValue({
+      ok: true,
+      text: () => Promise.resolve('swagger: "2.0"'),
+    });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterAll(() => {
+    delete process.env.USER_SERVICE_SPEC_VERSION;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)